docs/migration/introduction.md

```diff
@@ -27,6 +27,18 @@ If you encounter high gas fees when migrating LYXe to LYX, there could be severa
 - **Nonce-related**: Please re-try to migrate your LYXe using your browser wallet after **resetting your wallet activity**. Transaction problems might be related to the nonce not being updated correctly, letting the transaction fail, or raising its gas. Here you can find an example guide: [How to reset the activity on MetaMask](https://support.metamask.io/hc/en-us/articles/360015488891-How-to-clear-your-account-activity-reset-account).
 - **Gas-related**: If you only have a small amount of Ether to pay for the migration on Ethereum, MetaMask may estimate the gas a bit higher than your balance, causing very high gas fees of more than 0.5 ETH, instead of the regular 0.01 ETH. To solve the issue, please **make sure your wallet has enough Ether to pay for the transaction** at the current [transaction prices](https://etherscan.io/gastracker). You can also adjust the transaction speed to be on the lower end.
 
+:::info
+
+Some users managed to migrate with regular gas fees using a new browser like Microsoft Edge or Brave. The environment change will reset potential settings and cached data. If the high gas issue persists, please try to **import your private key or hardware wallet to a MetaMask wallet in a new desktop browser.**
```

Comment:
I suspect this to not be related to the browser but to the time.

I guess ppl had issue at T1 on browser A when gas fees were high on ethereum
When they try at T2 on browser B, maybe the activity on ethereum was lower.

Therefore the answer to the problem is because of trying at T2 and not because of trying on browser B.